test/functional/helpers/edgeInfo/domain.js

```diff
@@ -0,0 +1,21 @@
+const env = process.env.EDGE_ENV || "int";
+
+const domains = {
+  int: {
+    third: "edge-int.adobedc.net",
+    first: "firstparty-int.alloyio.com"
+  },
+  prod: {
+    third: "edge.adobedc.net",
+    first: "firstparty.alloyio.com"
+  }
+};
+
+export default {
+  getThirdParty() {
+    return domains[env].third;
+  },
```

Comment:
Any reason these need to be functions or could we just do `export default domains[env];`?

Reply:
No reason to be a function other than the need to overcomplicate them 😄 I'll update those as well.